extensions/zen-browser/src/actions/index.ts

```diff
@@ -1,31 +1,31 @@
 import { closeMainWindow, getPreferenceValues, popToRoot } from "@raycast/api";
 import { runAppleScript } from "run-applescript";
-import { Preferences, Tab } from "../interfaces";
 import { NOT_INSTALLED_MESSAGE, SEARCH_ENGINE } from "../constants";
+import { Preferences, Tab } from "../interfaces";
+import { getNewTabShortcut } from "../util";
 
 export async function openNewTab(queryText: string | null | undefined): Promise<boolean | string> {
   popToRoot();
   closeMainWindow({ clearRootSearch: true });
 
   const script = `
     tell application "Zen Browser"
-      activate
-      repeat while not frontmost
+    set savedClipboard to get the clipboard
+    set the clipboard to "${SEARCH_ENGINE[getPreferenceValues<Preferences>().searchEngine.toLowerCase()]}${queryText}"
```

Comment:
One thing I find strange about this is the text I search for or new tab opens in Google when my default search is DDG. I don't think that is related to your code since this existed before, but strange nevertheless.